applications/osb-portal/src/middleware/osbbackend.ts

```diff
@@ -19,7 +19,7 @@ const fetchWorkspacesAction = (): CallApiAction => {
   return ({
     type: 'api/fetchWorkspaces',
     payload: {
-      url: 'https://workspaces.cloudharness.metacell.us/api/workspace?page=1&per_page=2000',
+      url: '/api/workspaces/api/workspace?page=1&per_page=2000',
```

Comment:
@zsinnema do you think we can avoid /api repetition? My concern is that can easily led to confusion and possibly overlap with local api function names. Maybe something more explicit about we are asking for another microservice api like `/crossapi/workspace/api/...`